config/main.py

```diff
@@ -544,7 +546,11 @@ def reload(filename, yes, load_sysinfo):
         command = "{} -H -k {} --write-to-db".format(SONIC_CFGGEN_PATH, cfg_hwsku)
         run_command(command, display_cmd=True)
 
-    command = "{} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename)
+    if os.path.isfile('/etc/sonic/init_cfg.json'):
```

Comment:
Since you created a constant, please use it here, also:

```
    if os.path.isfile(INIT_CFG_FILE):
```

Reply:
Fixed.